File: ruby_related/naive_stack_sol.rb
### Write the class
class Stack

  def initialize
    # start new list, i.e. the array
    @list = []
  end

  def push(val)
    # push val to back
    #   of list
    @list.push(val)
  end

  def pop
    # return a val at the
    #   front of the list
    @list.pop
  end

end

File: javascripts/naive_stack_sol.js
// Write the constructor
function Queue() {
  // a list attribute
  this.list = [];
} 

Queue.prototype.push = function(val){
  // push a val into the back of the list
  this.list.push(val);
  return this;
};

Queue.prototype.pop = function(){
  // remove a val from the front of the list
  return this.list.pop()
};